README.md

```diff
@@ -50,30 +50,7 @@ db <- connect("laminlabs/cellxgene")
 db
 ```
 
-    <cellxgene>
-      Inherits from: <Instance>
-      Public:
-        Artifact: active binding
-        bionty: active binding
-        Collection: active binding
-        Feature: active binding
-        FeatureSet: active binding
-        FeatureValue: active binding
-        get_module: function (module_name) 
-        get_module_names: function () 
-        get_modules: function () 
-        initialize: function (settings, api, schema) 
-        Param: active binding
-        ParamValue: active binding
-        Run: active binding
-        Storage: active binding
-        Transform: active binding
-        ULabel: active binding
-        User: active binding
-      Private:
-        .api: API, R6
-        .module_classes: list
-        .settings: InstanceSettings, R6
+    cellxgene Instance(modules='c('core', 'bionty')')
```

Comment:
This should probably say something like `cellxgene Instance(modules=c('core', 'bionty'))`.

Given that there is currently no documentation on what you can or cannot do with an Instance class, could we shed some light on:

* The instance core classes
* An instance's extra modules (e.g. bionty, wetlab)
* A list of functions uses can call

Reply:
Or maybe this is something that should be part of something like `instance$describe()`?

Reply:
I've changed it to the longer format I was using for `Module` and `Registry`. I think that should show enough info now.